__tests__/endpoint.test.js

```diff
@@ -78,4 +71,32 @@ describe('Endpoint Tests', () => {
               });
           });
     })
+    describe('/api/articles/', () => {
+        test.only('200: response with an articles array of article object, which should contain all collumns except body, and be in data descending order', () => {
+            return request(app).get('/api/articles').expect(200)
+            .then(({body}) => {
+                console.log(body.data, 'in test')
+                expect(body.data).toHaveLength(13)
+                expect(Array.isArray(body.data))
```

Comment:
We don't need to explicitly check if the body is an array here, as the fact we are able to use a forEach on the response body, indicates that it is an array